src/code/graphalloc.c

```diff
@@ -0,0 +1,25 @@
+#include "global.h"
+
+Gfx* Graph_GfxPlusOne(Gfx* gfx) {
+    return gfx + 1;
+}
+
+Gfx* Graph_BranchDlist(Gfx* gfx, Gfx* dst) {
+    gSPBranchList(gfx, dst);
+    return dst;
+}
+
```

Comment:
ooo interesting that this is its own file. thinking about it, theres nothing about this that needs to be tied to "Graph" or graph thread, is there? I guess its gfx related and "graph" refers to graphics...

Basically im wondering if a different system name makes sense

Reply:
I see mm already has this file, is "graphalloc" made up or known from somewhere?

Reply:
My guess is made up and probably came from the fact these functions were in OoT's graph, but I'm not sure.
I wouldn't be opposed to a new system name, though I have ideas to rename the functions anyways.

Reply:
did you meant to say you dont have any ideas?
I dont really either rn tbh, but something to consider at a later time

Reply:
Ah, looks like AF has these functions in `gfxalloc.c`

Reply:
yeahhh I was thinking it should be gfx instead of graph at least. closer tied to Gfx than the graph thread itself.
Would love to see that change but doesnt have to be in this pr

Reply:
IMO it would be nice the change the filename now at least, but not necessarily the function names

Reply:
Went ahead and just made the changes now.